core/src/debug/RotatingEncryptableSink.cpp

```diff
@@ -125,6 +127,9 @@ namespace ledger {
 
         void RotatingEncryptableSink::_rotate() {
             auto resolver = _resolver.lock();
+			if (!resolver) {
```

Comment:
What happened with all your indentations ? it seems broken everywhere 😞

Reply:
Oh.... let me check